data/json/npcs/talk_tags.json

```diff
@@ -565,7 +583,122 @@
           "Look out!  A",
           "<swear!>, here comes a <swear>"
         ]
-    },{
+    },
+    {
+      "type": "snippet",
+      "category": "<general_danger>",
+      "text": [
+        "Look sharp!  Things are heating up.",
+        "Hey, <name_g>!  Hostiles inbound.",
```

Comment:
Suggestion: take "hey, <name_g>" out of this and the one below and instead make a snippet:
Hey, <name_g>!  <general_danger>

Reply:
Sure.